Sketch.java

```diff
@@ -1,36 +1,115 @@
 import processing.core.PApplet;
 
 public class Sketch extends PApplet {
-	
-	
+
+
+	/**
+  * Description: 
+  * Write a program that demonstrates a method with parameters and a return value
+  * @author: Tao A
+  */
+
+  int dimension = 500;  // set this for a global dimension value >= 100
+  
   /**
    * Called once at the beginning of execution, put your size all in this method
    */
   public void settings() {
 	// put your size call here
-    size(400, 400);
+    size(dimension, dimension);
   }
 
   /** 
    * Called once at the beginning of execution.  Add initial set up
    * values here i.e background, stroke, fill etc.
    */
   public void setup() {
-    background(210, 255, 173);
+    background(161, 185, 207);
+  }
+
+  /**
+    * this function returns the x / y values of the centre of the screen after drawing the object of the given size
+    * @param size  size of cross hair
+    * @return the x / y value for the object to be drawn in the middle of the screen
+    */
+  public int[] drawCrossHair(int size) {
+    fill(247, 2, 2);
+    int midpoint = dimension / 2;
+    rect(midpoint, midpoint,
+         size * dimension / 400, 3 * dimension / 400);
+    rect(midpoint + size / 2, midpoint - size / 2,
+         3 * dimension / 400, size * dimension / 400);
+
+    int[] location = new int[2];
+    location[0] = midpoint;
+    location[1] = midpoint;
+
+    // return the x / y value for the object to be drawn in the middle of the screen
+    return location;
   }
 
   /**
    * Called repeatedly, anything drawn to the screen goes here
    */
   public void draw() {
```

Comment:
the drawing of various objects should be done within their own method and called within the `draw()` method